pom.xml

```diff
@@ -189,7 +189,7 @@
         <module>xstream</module>
 
         <module>struts2</module>
-
+        <module>hibernate5</module>
   </modules>
 
-</project>
```

Comment:
@KevinGilmore  I think this is about parent pom.xml file. I've pulled, merged, resolved, commited and pushed. Let me know still if you are facing issues.